src/main/java/com/spotinst/sdkjava/model/SpotOceanEcsClusterService.java

```diff
@@ -0,0 +1,208 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.client.response.BaseServiceEmptyResponse;
+import com.spotinst.sdkjava.client.response.BaseSpotinstService;
+import com.spotinst.sdkjava.client.rest.*;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.api.ocean.ecs.ApiClusterLaunchSpecification;
+import com.spotinst.sdkjava.model.responses.ClusterEcsLaunchSpecApiResponse;
+import org.apache.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class SpotOceanEcsClusterService extends BaseSpotinstService {
+    public static ApiClusterLaunchSpecification createLaunchSpecification(
```

Comment:
Change function-name to createLaunchSpec and follow same for other methods in this class.